examples/xkafka/main.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+
+	"github.com/rs/xid"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	"github.com/gojekfarm/xrun"
+	"github.com/gojekfarm/xtools/xkafka"
+)
+
+const (
+	topic  = "xkafka-example"
+	broker = "localhost:9092"
+)
+
+func main() {
```

Comment:
Should we move this to `example_test.go`? That way it will be visible on pkg.go.dev also in the docs.